### PR TITLE
Meson build fixes

### PR DESCRIPTION
**Pull request by zeha**
Hello Karel,

I have started experimenting building util-linux with meson for Debian.
Here is a first PR for the necessary patches. I think they should all be good for upstream.

Debian builds fdisk,sfdisk,blkid statically, there I still have some work to do.

Note that I started off 2.38.1, but these patches are for master.

Would appreciate a review, thoughts, comments, ...
I haven't really used meson before.

Thanks,
Chris

### CODE REVIEW

**Comment by karelzak** [comment]
@t-8ch and @neheb, any objection? The meson changes seem fine to me.

File: meson.build [hunk 1]
```diff
@@ -23,6 +23,7 @@ docdir = join_paths(prefixdir, get_option('datadir'), 'doc', 'util-linux')
 mandir = join_paths(prefixdir, get_option('mandir'))
 runstatedir = '/run'
 execprefixdir = prefixdir
+sysconfdir = join_paths(prefixdir, get_option('sysconfdir'))
```
Comment (neheb) [inline meson.build:26]: You can use / instead of join_paths

**Comment by neheb** [comment]
Seems fine.

**Review by t-8ch — APPROVED** [review]

File: meson.build (hunk 1 above)
Comment (zeha) [reply meson.build:26]: thanks! for now i'd probably keep join_paths as the code around this line also uses it.

**Pull request merged**